backend_new/prisma/seed-helpers/household-member-factory.ts

```diff
@@ -0,0 +1,45 @@
+import { Prisma, YesNoEnum } from '@prisma/client';
+import { addressFactory } from './address-factory';
+import { randomAdjective, randomNoun } from './word-generator';
+import {
+  randomBirthDay,
+  randomBirthMonth,
+  randomBirthYear,
+} from './number-generator';
+import { randomBoolean } from './boolean-generator';
+
+export const householdMemberFactorySingle =
+  (): Prisma.HouseholdMemberCreateWithoutApplicationsInput => {
+    const firstName = randomNoun();
+    const lastName = randomNoun();
+    const randomYesNo = randomBoolean() === true ? YesNoEnum.yes : YesNoEnum.no;
+    return {
+      firstName: firstName,
+      middleName: randomNoun(),
+      lastName: lastName,
+      // Question: why are these strings?
```

Comment:
post launch of prisma we defo want to go back and clean these up to be better typed. For now though we're keeping things the way they are